mtav6.2.2/application.properties

```diff
@@ -0,0 +1,6 @@
+quarkus.kogito.devservices.enabled=false
+
+mta.url = ${MTA_URL:http://mta-ui.openshift-mta.svc.cluster.local:8080}
+quarkus.rest-client.mta_json.url = http://mta-ui.openshift-mta.svc.cluster.local:8080
```

Comment:
you should use `quarkus.rest-client.mta_json.url = ${mta.url}/hub` as in the original mta folder (same for the other application.properties)